app/controllers/search_controller.rb

```diff
@@ -38,13 +38,9 @@ def pattern_search
       return
     else
       flash_error(:runtime_invalid.t(type: :search, value: type.inspect))
-      redirect_back_or_default(
-        controller: :observations,
-        action: :index
-      )
+      redirect_back_or_default(:root)
```

Comment:
I think this should go to :root, which should vary with whether user is logged in.

Reply:
Agree